libs/blocks/reading-time/reading-time.css

```diff
@@ -12,3 +12,45 @@ main > .section > .content.reading-time > span {
 .center .reading-time > span {
   text-align: center;
 }
+
+main > .section > .content.inline-wrapper {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 8px 0 12px;
+}
+
+main > .section.center > .content.inline-wrapper {
+  justify-content: center;
+}
+
+main > .section > .content.inline-wrapper > .reading-time.inline {
+  margin: 0;
+  order: 1;
+  margin-left: 20px;
+}
+
+.reading-time.inline {
+  font-style: italic;
+  font-weight: 700;
```

Comment:
Different text styling for inline variant?

Reply:
Fixed, made all variants the same